test/FunctionalTests/Microsoft.OData.Core.Tests/ValidationHelper.cs

```diff
@@ -0,0 +1,104 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="ValidationHelper.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.OData.Tests
+{
+    public static class ValidationHelper
+    {
+        /// <summary>
+        /// This method takes 2 generic types and see if one is the deep copy  of other
+        /// </summary>
+        /// <typeparam name="T">Method accepts Generic type</typeparam>
+        /// <param name="obj">Object to be compared with</param>
+        /// <param name="copy">Copy object to compare</param>
+        /// <returns>Returns a list of differences</returns>
+        public static List<string> GetDifferences<T>(T obj, T copy)
+        {
+            var diff = new List<string>();
+            EvaluateDifferences(obj, copy, diff);
+
+            return diff;
+        }
+
+
+        /// <summary>
+        /// To evaluate the differences of 2 generic types, called recursively
+        /// </summary>
+        /// <typeparam name="T">Method accepts Generic type</typeparam>
+        /// <param name="obj">Object to be compared with</param>
+        /// <param name="copy">Copy object to compare</param>
+        /// <param name="diff">List Of Differences</param>
+        /// <param name="objName">Optional parameter for objectname, to be used in differences string, default empty string</param>
+        /// <param name="isCollection">Optional parameter to see if its a collection, default false</param>
+        private static void EvaluateDifferences<T>(T obj, T copy, List<string> diff,string objName ="", bool isCollection =false)

```

Comment:
>,string objName ="" [](start = 83, length = 19)

add the whitespace correctly. for example:
add whitespace between the parameter, add the whitespace before and after "="?